pkg/resolver/static_responses.go

```diff
@@ -39,7 +39,7 @@ func init() {
 
 	r = &Response{
 		Status:  http.StatusInternalServerError,
-		Message: fmt.Sprintf("Could not fetch link info: Response too large (>%dMB)", MaxContentLength/1024/1024),
+		Message: fmt.Sprintf("Could not fetch link info: Response too large (>%dMB)", GetMaxContentLength()/1024/1024),
```

Comment:
Since you now return megabyes in your function, there is no need to divide, you can write just `GetMaxContentLength()`

Reply:
If you choose to do the above and multiply in the function, these divide calls can stay